examples/rbac/server/feature_repo/example_repo.py

```diff
@@ -0,0 +1,131 @@
+# This is an example feature definition file
```

Comment:
Don't we want to add permissions as well to the example?

Reply:
aded please take look again thanks